spec/services/fees/charge_service_spec.rb

```diff
@@ -67,6 +66,71 @@
         )
       end
 
```

Comment:
Can we maybe add one case here with combination of `grouped_by` and `adjusted_fee`?